### ISSUE TITLE
Additional border between card-header and list-group-flush in bs 4.5.

### ISSUE THREAD

**Issue by brettryan** [twbs/bootstrap#30983]
Before opening:

As of bootstrap v4.5 adding a `list-group-flush` to a card will now result in an inherited border between `card-header` and the `list-group`.

This is evident in the two examples comparing [4.4](https://getbootstrap.com/docs/4.4/components/card/#list-groups) and [4.5](https://getbootstrap.com/docs/4.5/components/card/#list-groups):

Note the markup from the example as follows:

```html
<div class="card" style="width: 18rem;">
  <ul class="list-group list-group-flush">
    <li class="list-group-item">Cras justo odio</li>
    <li class="list-group-item">Dapibus ac facilisis in</li>
    <li class="list-group-item">Vestibulum at eros</li>
  </ul>
</div>
```

This has been confirmed with the following browsers:

* Safari: Version 13.1 (15609.1.20.111.8)
* Fx: 75.0 (64-bit)
* Chrome: 83.0.4103.61 (Official Build) (64-bit)

**Comment by XhmikosR** [comment]
@twbs/css-review isn't this already fixed in v4-dev?

**Comment by ffoodd** [comment]
Fixed by #30808, duplicate of #30793 — @brettryan this'll be fixed in the next v4 :)

**Comment by brettryan** [comment]
Thanks so much guys. :) Sorry for creating the duplicate, I did search but my search skills not so good I guess lol.
